### ISSUE TITLE
ACO - C - 1.1 Transitions function

### ISSUE THREAD

**Issue by madammann** [pippowell/DiffEv#45]
Performs calculations on which node to visit next with which probability.
Returns _nodes_ and _probabilities_.
HINT: You may use _self.neighbours()_ function.

**Comment by madammann** [comment]
Hey Fabienne,
I see that you have moved this issue to "Review in progress", however, I cannot see your code pushed to any new branch.
Make sure to upload it and also post a merge request so your code can be reviewed.
If you need help with that, just message me :)
Best,
Marlon

**Comment by FaKock** [comment]
Hey Marlon,
that was the problem I mentioned in the chat. I accidentally pushed it directly on the main branch since I seemed to have a local origin problem for the branch. I think that was a problem of gitkraken... Should I push it again on a new branch with a merge request?
Best,
Fabienne

Update:
I created a new branch and added some spacing to create a proper pull request that can be reviewed. The function is just on the main antcolony branch already, so changes must overwrite the main antcolony branch.
